ckan/logic/action/create.py

```diff
@@ -724,7 +724,7 @@ def _group_or_org_create(context: Context,
     group_type = data_dict.get('type', 'organization' if is_org else 'group')
     group_plugin = lib_plugins.lookup_group_plugin(group_type)
     try:
-        schema: Schema = group_plugin.form_to_db_schema_options({
+        schema: Schema = getattr(group_plugin, "form_to_db_schema_options")({
```

Comment:
Just out of curiosity, why are these calls changed?

Reply:
Here `group_plugin` is an instance of `IGroupForm` according to the type signature of functions. This interface no longer has `form_to_db_schema_options` and the whole try/except block that wraps the current change exists for backward compatibility.

With a simplified implementation of the plugin system, the type checker now can detect this issue and report a call to the non-existing method `form_to_db_schema_options`. So I have to either add the `type: ignore` directive or modify the line to verify, that I know what am I doing. 

I decided to use getattr, because unlike `type: ignore`, it highlights the unsafe expression and now it's more clear, what exactly can cause an AttributeError that we are catching in try/except